### ISSUE TITLE
Badges API

### ISSUE THREAD

**Issue by yishaigalatzer** [NuGet/NuGetGallery#2889]
ported from: https://github.com/NuGet/Home/issues/2113

Currently using GitHub badges for my projects: https://github.com/bchavez/RethinkDb.Driver

The service I'm using to render badges is http://shields.io/. The problem is shields.io constantly offline or inaccessible. I've had so many problems getting NuGet metadata for the project.

![screen603](https://cloud.githubusercontent.com/assets/478118/13060478/a5020502-d3e6-11e5-90a8-c944d208ae10.png)

Would be nice if NuGet provided an API to render badges directly:
- Total Downloads.
- Download Count of current version.
- Last published date.

Thanks,
Brian

CC @bchavez

**Comment by bchavez** [comment]
Thank you. For comparison, here's what it looks like when the badges when they actually work:

![screen608](https://cloud.githubusercontent.com/assets/478118/13089948/05730410-d4a8-11e5-90ac-fc518af0e4d1.png)

**Comment by maartenba** [comment]
@yishaigalatzer is there a designer we can talk to to create the badges?

**Comment by robertmuehsig** [comment]
Should the NuGetGallery use the NuGet V3 itself to expose such badges? 
I have a similar (but simpler idea) to provide a simple Atom/RSS feed for packages - back in the old days (2 years ago or so) I tried it by accessing the database via EF, but I was told I shouldn't do it this way and wait for a more stable api, which doesn't hammer the database.

**Comment by maartenba** [comment]
For the badges, the gallery can make use of the search service (much like what the package details page does to check if the package was indexed or not)

**Comment by skofman1** [comment]
Is this still relevant? If so, please reopen.

**Comment by bchavez** [comment]
Yes, it's still relevant. I would reopen but I don't have that option here.

**Comment by skofman1** [comment]
@bchavez , can you please provide more details? Is http://shields.io/ continues to be offline frequently?

**Comment by bchavez** [comment]
@skofman1 , http://shields.io uptime has improved since we made this issue. However, **shields.io** has reduced the number of **NuGet** badges that it supports. I do not know the reasoning behind the reduced **NuGet** support.

Originally, we could obtain the ***download total*** :1234: badge ([seen here](https://github.com/NuGet/NuGetGallery/issues/2889#issuecomment-184861961)), but the ***download total*** badge is not available from http://shields.io anymore. Now, we've only got the **NuGet** package :package: :label: version number.

:cry:

**Comment by dustinmoris** [comment]
May I suggest to use [BuildStats.info](https://buildstats.info) for the NuGet badge? It's already widely used for many .NET projects hosted on GitHub (e.g. see the [NUnit](https://github.com/nunit/nunit) README).

**Comment by HypsyNZ** [comment]
Really I have to use a third party for badges in 2022

**Comment by martincostello** [comment]
BuildStats.info has been shut down now 😞

Looks like shields.io now supports both [versions](https://shields.io/badges/nu-get-version) and [downloads](https://shields.io/badges/nu-get-downloads) for NuGet packages.
